wellregistry/registry/models.py

```diff
@@ -199,7 +199,7 @@ class MonitoringLocation(models.Model):
     alt_method = models.CharField(max_length=300, blank=True, verbose_name='Altitude method')
     alt_acy = models.CharField(max_length=300, blank=True, verbose_name='Altitude accuracy')
 
-    well_depth = ArbitraryDecimalFields(null=True, blank=False)
+    well_depth = ArbitraryDecimalFields(null=True, blank=True)
```

Comment:
I would like to understand the relationship of the blank setting and None. If we have time in the sprint it might be good to research this.

Reply:
agreed